buildenv/jenkins/common/pipeline-functions

```diff
@@ -28,29 +28,46 @@
     // if not set, sniff the remote repositories references
 
     stage ('Sniff Repos') {
-        dir('openjdk') {
+        def description = ''
+
+        if (OPENJDK_SHA instanceof Map && OPENJDK_REPO instanceof Map && OPENJDK_BRANCH instanceof Map) {
```

Comment:
Use of `instanceof` will cause an error on use `Scripts not permitted to use method java.lang.Class isInstance java.lang.Object.` without approving the signature.  Is this something that can be reworked without?  Is it a big deal for approving?

Reply:
We'll need to approve it (process script approval). In addition to it will need approval for addAll and intersection.